packages/react-ui-components/src/stories/Components/Loader.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvLoader } from '../../components';
+
+KvLoader.displayName = 'KvLoader';
+
+export default {
+	title: "Components/Loader",
+	component: 'kv-loader',
+	argTypes: {
+		isLoading: { type: "boolean" },
+		hasOverlay: { type: "boolean" }
+	},
+	parameters: {
+		notes: require('../../../../ui-components/src/components/loader/readme.md')
```

Comment:
please rebase with dev branch and update to `require('@ui-notes/loader/readme.md')`